doc/manual/source/interaction/geopackage-import-export.rst

```diff
@@ -0,0 +1,174 @@
+GeoPackage import and export Web-API 
+====================================
+
+`GeoPackage <http://www.geopackage.org/>`_ is an open, standards-based, platform-independent, portable, self-describing, compact format for transferring geospatial information.
+
+
+The GeoPackage import-export GeoGig Web-API allows to download a repository snapshot or a subset of it as a GeoPackage file, and to upload a GeoPackage file and import its vector layers into the GeoGig repository.
+
+
+GeoPackage export
+-----------------
+
+Exports a repository snapshot or a subset of it as a GeoPackage file.
+
+A repository snaphot is the data (layers and their features) addressable from a certain commit.
+
+::
+
+   GET /<repo>/export[.xml|.json]?format=gpkg[&root=<refspec>][&path=<layerName>[,<layerName>]+][&bbox=<boundingBox>][&interchange=<true|false>]
+
+
+Parameters
+^^^^^^^^^^
+
+**format:**
+Mandatory. Must be ``gpkg`` (case insensitive). Defines the output format of the export operation.
+   
+**root:**
+Optional. Defaults to ``HEAD``. ref spec that resolves to the root tree from
```

Comment:
The later, the period is outside the double quotes, though I see how it can be confusing. Changing to Defaults to ``HEAD``. The ref spec...